### DISCUSSION TITLE
Paired with amp but can't get results from Tonecloud search

### DISCUSSION THREAD

**Discussion by itm1960** [soundshed/soundshed-app#67] [Q&A]
Thanks for a great app. 
I've just installed it, and managed to pair it with the Spark Amp, but when I try to search Tone Cloud for anything I never get any results. Searching from the Spark app on my phone works fine.
Any idea why the search function isn't working in Soundshed?

**Comment by webprofusion-chrisc** [comment]
Windows or macOS, and what OS versions?

If you try https://app.soundshed.com/ does it work for you searching tones?

**Reply by itm1960** [reply]
Sorry I should have said - Windows 10 x64 22H2.
Yes, tone searching at https://app.soundshed.com/ does work (although it seems to return different results to equivalent searches on the PG Spark app). 
I also wasn't sure whether favourites were persisted between sessions (or at all?)

**Reply by webprofusion-chrisc** [reply]
If you star a tone it is saved in a list and persisted between sessions, if that's not happening it would suggest a bug.

I can configure current app installed on Windows 11 is returning tone search results for me. Do you have anything that could be blocking outgoing https requests like malware protection software etc?

**Reply by godtaf** [reply]
I tried different web browsers two different Os's and installed application.  The search doesn't work very well.  Some things work but others don't.

**Reply by webprofusion-chrisc** [reply]
@godtaf the FX and amp definitions haven't been updated in a few years, they were fine for the original Spark 40 but there's likely to be incompatibility for any new fx or amp models. The actual search results seem to work ok for me but depending on what hardware you are trying it with results may vary, I've only got a spark mini currently.

**Reply by godtaf** [reply]
Thanks.  I don't understand I didn't connect at an amp I used the web app and a installed app in Linux. Try go to: https://app.soundshed.com/#/tones.  Search for "Sunshine for your love".  The results are really odd.  This is the same results I tested on Linux, Windows, web browsers connected to an amp or not.  All the same.  Then searched for "acdc hells bells"  That works. But "i love rock n roll jona jet" ,"jet are you gonna be my girl" & "lenny kravitz are you gonna go my way"  doesn't.  So the results are different if connect to an amp? I can try again but with my test it's the same.  Thanks Just trying to help.

**Reply by webprofusion-chrisc** [reply]
Yeah I don't know, when you search Tone Cloud it passed your search as a keyword to the positive grid tone cloud and it decided how tones do or don't match, so it should be much the same results as if you used the PG Spark app on a phone.

**Reply by godtaf** [reply]
Thanks.  That's what I thought.  I could see the phone app different but say15%. The one you want is number 6 in the list not 2. Something is off but what? I will look into some more Thanks.

**Comment by itm1960** [comment]
I only use the default Windows Defender AV - I don't have any other AV software installed. 
I've also just tried explicitly allowing the Soundshed app through the Windows Firewall, but it didn't help.
???
